packages/cypher-builder/src/clauses/sub-clauses/Delete.ts

```diff
@@ -19,10 +19,9 @@
 
 import { CypherASTNode } from "../../CypherASTNode";
 import type { CypherEnvironment } from "../../Environment";
-import type { NodeRef } from "../../references/NodeRef";
-import type { RelationshipRef } from "../../references/RelationshipRef";
+import type { Variable } from "../../references/Variable";
 
-export type DeleteInput = Array<NodeRef | RelationshipRef>;
+export type DeleteInput = Array<Variable>;
```

Comment:
I believe this should be:
```suggestion
export type DeleteInput = Array<NodeRef | RelationshipRef | Variable>;
```

Reply:
Good catch